server/document_index.worker.js

```diff
@@ -41,17 +39,25 @@ function postDocumentInfo(filePath, changeType) {
 }
 
 const watcher = chokidar.watch(
-  [CONTENT_ROOT, CONTENT_ARCHIVE_ROOT]
-    .filter(Boolean)
-    .map((root) => path.join(root, "**", "*.html"))
+  // For now, brutally hardcode it to only the 'en-us' folders
+  // until we have a resolution on L10n.
+  ROOTS.filter(Boolean).map((root) => path.join(root, "en-us", "**", "*.html"))
 );
 
+let countWatchedFiles = 0;
+const t0 = new Date();
 watcher.on("ready", () => {
   postEvent("ready");
+  const took = new Date() - t0;
+  console.log(
+    `Watching over ${countWatchedFiles.toLocaleString()} files. ` +
```

Comment:
I put this in for my own sanity as I was making this fix. Then, I thought it might be useful if only it can also measure how long that took.